product_docs/docs/pgd/5/durability/group-commit.mdx

```diff
@@ -76,9 +76,7 @@ to commit something. This approach requires application changes to use
 the CAMO transaction protocol to work correctly, as the application is in some way
 part of the consensus. For more on this approach, see [CAMO](camo).
 
-The `raft` option is for backwards compatibility only. It uses PGD's built-in Raft
-consensus for commit decisions. It is slow and only useful with eager conflict
-resolution.
+The `raft` decision uses PGDs built-in raft consensus for commit decisions. It can have low performance and is currently only required when doing eager conflict resolution with an ALL commit scope group.
 
```

Comment:
It can have low performance and is currently only required with an ALL commit scope group. This means when there is use of ALL, the commit_decision needs to be 'raft'.

Reply:
The `raft` decision uses PGDs built-in raft consensus for commit decisions. It can have low performance and is currently only required when using GROUP COMMIT with an ALL commit scope group.

Reply:
yes. looks good.